n6001/hardware/ofs_n6001/build/scripts/run.sh

```diff
@@ -192,11 +188,11 @@ if [ ! -f fpga.bin ]; then
 fi
 
 #copy fpga.bin to parent directory so aoc flow can find it
-cp fpga.bin ../
-cp acl_quartus_report.txt ../
+cp fpga.bin $RELATIVE_KERNEL_BUILD_PATH_TO_HERE/
```

Comment:
Still need to copy these files from the Quartus-build-directory (syn_top) to the kernel-build-directory (ofs_n6001).